app/command_utils.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_APP_COMMAND_UTILS_H_
+#define BRAVE_APP_COMMAND_UTILS_H_
+
+#include <string>
+#include <vector>
+
+namespace commands {
+
+std::vector<uint16_t> GetCommands();
```

Comment:
nit: I believe this could return `const &`

Reply:
Whoops, I actually thought I'd done that! Nice catch